lib/plugins/aws/customResources/resources/utils.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const AWS = require('aws-sdk');
+const _ = require('lodash');
```

Comment:
We should not introduce any dependencies here (note this is packaged for lambda, and we want to avoid `npm install`)

Reply:
Yeah. Will remove the usage of `lodash`.